fluidBuild.config.cjs

```diff
@@ -71,8 +71,8 @@ module.exports = {
 			dependsOn: ["build:esnext"],
 			script: true,
 		},
-		"build:docs": ["tsc"],
-		"ci:build:docs": ["tsc"],
+		"build:docs": ["tsc", "build:esnext"],
```

Comment:
Non-ci should probably depend on "api" if there is a change.
packages that have build:docs and api* script should have `"build:docs": "fluid-build . --task api"` which fluid build should figure out.

Reply:
I see that @fluid-experimental/oldest-client-observer's dependencies already have build:docs depending on api-extractor:esnext which is configured globally to depend on build:esnext.
Also @fluid-experimental/oldest-client-observer ci:build:docs is based on cjs (according to its api-extractor.json). I don't know where these errors are coming from.